daml.yaml

```diff
@@ -1,4 +1,4 @@
-sdk-version: 0.13.55
+sdk-version: 1.2.0
 name: dablchat
 version: 1.7.2
```

Comment:
given that this has a daml change it will need a minor version upgrade. (ie 1.8.0)
There are unfortunately 4 places where you have to do this.
- package.json for the ui
- setup.py (x2) for the bots
- daml.yaml for the model